tests/unit/lib/test_opensearch_base_charm.py

```diff
@@ -289,6 +289,30 @@ def test_on_update_status(self, _, cert_expiration_remaining_hours, _stop_opense
             self.charm.on.update_status.emit()
             self.assertTrue(isinstance(self.harness.model.unit.status, BlockedStatus))
 
+    @patch(f"{BASE_LIB_PATH}.opensearch_tls.OpenSearchTLS.store_admin_tls_secrets_if_applies")
+    @patch(f"{BASE_CHARM_CLASS}.is_admin_user_configured")
+    @patch(f"{BASE_LIB_PATH}.opensearch_tls.OpenSearchTLS.is_fully_configured")
+    @patch(f"{BASE_LIB_PATH}.opensearch_tls.OpenSearchTLS.reload_tls_certificates")
+    def test_on_tls_conf_set(
+        self,
```

Comment:
This function could have a name like `test_tls_configured_without_restart()` or so. As it _is_ verifying that!

Reply:
Done